api/helpers.js

```diff
@@ -17,7 +17,7 @@ const reformatDate = (date, oldFormat, newFormat) =>
 const toBritishDateFormat = (date) => reformatDate(date, formatInternational, formatBritish);
 const toInternationalDateFormat = (date) => reformatDate(date, formatBritish, formatInternational);
 
-const buildBirthParams = (attrs) => _.pickBy({
+const buildParams = (attrs) => _.pickBy({
```

Comment:
Why was this renamed?

Reply:
Ah, I think I thought that was one of the birth changes I had made... I switch that back...